go/ct/spc/specification.go

```diff
@@ -1347,6 +1347,27 @@ func getAllRules() []Rule {
 
 	rules = append(rules, callOp()...)
 
+	// --- SELFDESTRUCT ---
+
+	//                                          revision   warm  destColdCost accCreatFee  refund
+	rules = append(rules, nonStaticSelfDestOp(R07_Istanbul, false, vm.Gas(0), vm.Gas(25000), vm.Gas(24000))...)
+	rules = append(rules, nonStaticSelfDestOp(R07_Istanbul, true, vm.Gas(0), vm.Gas(0), vm.Gas(24000))...)
+	rules = append(rules, nonStaticSelfDestOp(R09_Berlin, false, vm.Gas(2600), vm.Gas(25000), vm.Gas(24000))...)
+	rules = append(rules, nonStaticSelfDestOp(R09_Berlin, true, vm.Gas(0), vm.Gas(0), vm.Gas(24000))...)
+	rules = append(rules, nonStaticSelfDestOp(R10_London, false, vm.Gas(2600), vm.Gas(25000), vm.Gas(0))...)
+	rules = append(rules, nonStaticSelfDestOp(R10_London, true, vm.Gas(0), vm.Gas(0), vm.Gas(0))...)
+
+	rules = append(rules, rulesFor(instruction{
+		op:        SELFDESTRUCT,
+		name:      "_staticcall",
+		staticGas: 5000,
+		conditions: []Condition{
+			Eq(ReadOnly(), true),
+			AnyKnownRevision(),
+		},
+		effect: FailEffect().Apply,
```

Comment:
```suggestion
		effect: FailEffect(),
```

Reply:
`FailEffect()` returns an `Effect`  which is not a `func(*state)`, hence I need to call the `.Apply` method, which does meet the required interface.